spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/RabbitListenerConfigUtils.java

```diff
@@ -36,4 +36,29 @@
 	public static final String RABBIT_LISTENER_ENDPOINT_REGISTRY_BEAN_NAME =
 			"org.springframework.amqp.rabbit.config.internalRabbitListenerEndpointRegistry";
 
+	/**
+	 * The bean name of the default RabbitAdmin.
+	 */
+	public static final String RABBIT_ADMIN_BEAN_NAME = "amqpAdmin";
+
+	/**
+	 * The bean name of the default ConnectionFactory.
+	 */
+	public static final String RABBIT_CONNECTION_FACTORY_BEAN_NAME = "rabbitConnectionFactory";
+
+	/**
+	 * The default property to enable/disable MultiRabbit processing.
+	 */
+	public static final String MULTI_RABBIT_ENABLED_PROPERTY = "spring.multirabbitmq.enabled";
```

Comment:
Shouldn't this be aligned with boot property naming? `spring.rabbitmq.multi-cluster-enabled` or similar, and added to `RabbitProperties` (`multiClusterEnabled`)?

Otherwise users will get warnings in their IDE about an unknown property.

Reply:
I will provide boot part with that.
The reason I initially didn't add it under `spring.rabbitmq` (and therefore at `RabbitProperties`), is because the attribute would also be replicated to all brokers under `spring.multirabbitmq.connections.[]` which might cause confusion.
E.g.: `spring.multirabbitmq.connections.brokerA.multi-cluster-enabled`

What would you think about `spring.multirabbitmq.multi-cluster-enabled` (and therefore at `MultiRabbitProperties`)?

```
@ConfigurationProperties(prefix = "spring.multirabbitmq")
public class MultiRabbitProperties {

	/**
	 * A flag to enable/disable MultiRabbit processing.
	 */
	@Value("${" + RabbitListenerConfigUtils.MULTI_RABBIT_ENABLED_PROPERTY + "}")
	private boolean multiClusterEnabled;

	/**
	 * A map representing the RabbitProperties of all available brokers.
	 */
	private Map<String, RabbitProperties> connections = new HashMap<>();

	public Map<String, RabbitProperties> getConnections() {
		return this.connections;
	}

	public void setConnections(@Nullable final Map<String, RabbitProperties> connections) {
		this.connections = Optional.ofNullable(connections).orElse(new HashMap<>());
	}

}
```

Reply:
Ah, yes; makes sense. 

Maybe just `spring.multirabbitmq.enabled`?

Thanks.